src/SNTV2.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.19;
+
+import { MiniMeToken } from "@vacp2p/minime/contracts/MiniMeToken.sol";
+
+contract SNTV2 is MiniMeToken {
+    constructor(
+        string memory _tokenName,
+        uint8 _decimalUnits,
+        string memory _tokenSymbol
+    )
+        MiniMeToken(MiniMeToken(payable(address(0))), 0, _tokenName, _decimalUnits, _tokenSymbol, true)
```

Comment:
I've hardcoded some of the parameters here as I don't think it's likely we'll ever want to deploy an SNT token that is not transferable or needs a parent block snapshot, or parent token.

@3esmit lemme know if you think this should be changed.